### ISSUE TITLE
Can you provide a patch for the MPD distribution with SACD ISO support?

### ISSUE THREAD

**Issue by montaintop** [lintweaker/mpd-dsd-019#21]
Dear lintweaker

Maxim V.Anisiutkin created a MPD distribution with SACD ISO direct support. it would be perfect if we can have a native DSD support together with SACD ISO support. can you create a patch for that distribution?

GIT link attached below. greatly appreciated. 
http://git.musicpd.org/cgit/manisiutkin/mpd.git

**Comment by hicaoc** [comment]
mpd 0.20 support native dsd 。 Maxim V.Anisiutkin's  last version also support ！
I forked it
https://github.com/hicaoc/MPD.git
./configure --enable-sacdiso

**Comment by montaintop** [comment]
you are the man! btw, can u update you fork? seems quite a few update from max recently:) forgive me if I am asking a stupid quesiton

**Comment by hicaoc** [comment]
Maxim V.Anisiutkin few days before the merger master version of these updates should include

http://git.musicpd.org/cgit/manisiutkin/mpd.git/

**Comment by montaintop** [comment]
Thanks for the quick response

u mean his repo will work for native dsd support already ? or yours?
 https://github.com/hicaoc/mpd-sacd-nativeDSD

**Comment by hicaoc** [comment]
yes ，His repo work for native dsd already 。 My version just for my DAC revised some places , for example , the sample_rate DSD_U32_BE mode ,
If you play native dsd also speed becomes faster or have the noise problem , you can try my version

**Comment by ronnyzhao** [comment]
Hello, I want to play DSD file in Native mode according to this git way：
1.update kernel quirks file:   try DSD_U32_BE and  DSD_U32_LE
2. alsa-lib 1.1.16
3. mpd: 0.19.19, 0.20.19， all try
but, MY DAC display PCM 384KHZ,
and, play noise。 my mpd.conf config: dop "no", dsd_native "yes" and another type is 2.
I remove kernel quirks the code added myself for my DAC.
then play DSD128 ,  MY DAC still display PCM 384khz.
play normally。
How to do it , if i want to play DSD in native mode.
please give me help, thanks very.

**Comment by ronnyzhao** [comment]
another issue:
mpd play DSD64 smoothly, but DSD128 not,  CPU: cpu0 100% too high
so how to solution it? thanks!!!!
